utils/rowgroup/rowgroup.h

```diff
@@ -1950,6 +2018,10 @@ inline void copyRow(const Row& in, Row* out, uint32_t colCount)
     {
       out->setLongDoubleField(in.getLongDoubleField(i), i);
     }
+    else if (UNLIKELY(in.getColTypes()[i] == execplan::CalpontSystemCatalog::DOUBLE))
+    {
+      out->setDoubleField(in.getDoubleField(i), i);
```

Comment:
Ehhm. Imagine in Row has NULL double value that is an invalid double from the perspective of IEEE-754. IMHO the integer NULL magic will be saturated when it is implicitly casted to double.

Reply:
If this is not the case then where is if for FLOAT?

Reply:
I think you may be right here. I'll test your suggestion.

Still, no tests are failing with that! What does that tell us?

Reply:
Upon inspection the setDoubleField(getDoubleField(...),...) can be safely handled by last leg of the if...else if...else conditional.

Reply:
This tells there are no relevant tests.